tests/test_reader.py

```diff
@@ -26,6 +26,10 @@
         #     "pro",
         #     id="Phi .pro reader",
         # ),
+        pytest.param(
```

Comment:
on line 19 `reader_dir` is confusing for the outside reader. You can rename it, like `sub_data_dir`.

Reply:
renamed